R/explore_summary.R

```diff
@@ -1,13 +1,31 @@
 #' Provides basic summary statistics and wholistic
 #' information about all the features of provided data
 #'
-#' @param data dataframe
+#' @param df dataframe (clean)
 #'
-#' @return dataframe
-#' @export
+#' @return a summary dataframe
 #'
 #' @examples
 #' explore_summary(iris)
-explore_summary <- function(data) {
+#'
+#'
+library("testthat")
+library("matrixStats")
+explore_summary <- function(df) {
+  # retrieve only numeric variables
+  num_index <- data.frame("index"=which(sapply(colnames(df), function(x) is.numeric(df[, x]))))
+  index <- c()
+  for(i in(1:nrow(num_index))){
+    index <- c(index, num_index[i,1])
+  }
+  num_data <- data[index]
```

Comment:
I am not able to run the code. This line is creating error. Please review and modify

Reply:
It runs on my device... Let me double check...

Reply:
The problem was fixed by https://github.com/UBC-MDS/xplrrr/pull/15/commits/9552a849eaa8be1b4e55bbc0e66f418bf81220ac